Packs/Silverfort/Integrations/Silverfort/Silverfort.py

```diff
@@ -1,3 +1,5 @@
+import time
+import jwt
```

Comment:
does Demisto support this pkg?

Reply:
@DanielZiv-SF  - Not sure you saw the discussion in the DFIR channel (`#silverfort-partner`) but we mentioned there that you need to change the Docker image to one that has `jwt` package included. We mentioned that you may use the `demisto/pyjwt3` Docker image in this case.
To do this, set the `script.dockerimage: demisto/pyjwt3:1.0.0.29644` in `Packs/Silverfort/Integrations/Silverfort/Silverfort.yml`.